scripts/contiv-vpp-bug-report.sh

```diff
@@ -104,13 +104,66 @@ get_shell_data_ssh() {
     ssh "$SSH_USER@$NODE_NAME" "${SSH_OPTS[@]}" "$@" > "$log" || true
 }
 
+read_shell_data_local() {
+    "$@" || true
+}
+
+read_shell_data_ssh() {
+    ssh "$SSH_USER@$NODE_NAME" "${SSH_OPTS[@]}" "$@" || true
+}
+
+read_node_shell_data() {
+    if [ "$USE_SSH" = "1" ]
+    then
+        read_shell_data_ssh $@
+    else
+        read_shell_data_local $@
+    fi
+}
+
 get_k8s_data() {
     log="$1"
     shift
     echo " - kubectl $*"
     master_kubectl "$@"> "$log"
 }
 
+save_container_nw_report() {
+    # generates Docker container network report for the $NODE_NAME into the $CONTAINER_NW_REPORT_FILE
+    # for each docker container, prints out its interfaces, routing table and ARP table
+    echo " - ${CONTAINER_NW_REPORT_FILE}"
+
+    # get ID + NAME of each container except the pause containers
+    CONTAINERS_TXT=$(read_node_shell_data "sudo docker ps --format '{{.ID}} {{.Names}}' | grep -v pause")
+    while read -r line; do CONTAINERS+=( "$line" ); done <<< "$CONTAINERS_TXT"
```

Comment:
To be consistent with the other lines in this file, please split it out like so.  Also, please use lower case for local function variables.
```
containers=()
while read -r line
do
    containers+=( "$line" )
done <<< "$containers_txt"
```
The top declaration (which could also be written 'declare -a containers') is needed in case there are no matching lines, otherwise due to the 'set -u' at the top of the file, trying to use the containers variable below will blow up with an uninitialized variable error.